src/main/java/Event.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Represents a task to be completed.
+ * A <code>Event</code> object contains description of the task e.g., <code>meeting</code> and timing that the task will
+ * occur e.g., <code>Sunday 10am</code>.
+ */
+public class Event extends Task {
+    protected String timing;
+
+    public Event(String command, String timing) {
+        super(command);
+        this.timing = timing;
+    }
+
+    /**
+     * Returns whether a task is marked done in int form instead of symbol form, to be displayed in the file
+     * saved in hard disk.
+     *
+     * @return 1 if task is marked done, or 0 if the task is marked undone.
+     */
+    public int getDoneInt() {
```

Comment:
Can define this method in Task instead, since all 3 types of Task have this method